functional-tests/README.md

```diff
@@ -34,13 +35,14 @@ Follow the `.env-example` template to create an `.env` file under `/functional-t
 ## Run tests with Gradle
 
 The following commands will launch the tests with the individual browsers.
+They must be run in the functional test directory.
```

Comment:
This seems redundant

Reply:
`run test`  runs in the base directory.  This is to differentiate it